openssh/README.md

```diff
@@ -0,0 +1,100 @@
+# openssh
+
+## Maintainers
+
+The Habitat Maintainers: <humans@habitat.sh>
+
+## Usage
+
+## Testing
+
+### Build the openssh core plan
+
+Enter a habitat studio
+
+```bash
+hab pkg build .
+```
+
+Exit habitat studio
+
+### Build and install the test plan
+
+Enter another habitat studio
+
+```bash
+# stop the habitat supervisor
+hab sup term
+
+# install the openssh artifact
+source /src/results/last_build.env
+hab pkg install -b /src/results/${pkg_artifact}
+
+# add some required software before building the openssh_test
+hab pkg install -b core/hab-studio
+hab pkg install -b core/busybox
+hab pkg install -b chef/inspec
+
+# build openssh_test CD'ing into the test directory first
+( cd /src/test && hab pkg build . )
+
+# install openssh_test
+source /src/test/results/last_build.env
+hab pkg install -b /src/test/results/${pkg_artifact}
+
+hab sup run &
+sleep 20
+
+hab svc load ${pkg_origin}/${pkg_name}
+```
+
+Remain in the same habitat studio for the next section
+
+### Run the tests
+
+Create a yml version of the last_build.env for inspec testing and then run the tests
+
+```bash
+sed -n "2,$ p" "/src/results/last_build.env" | sed "s/=/: '/g" | sed "s/$/'/g" > "/src/results/last_build.yml"
```

Comment:
In the sed PR I avoided needing to pass any YAML by instead having the InSpec tests figure out the pkg_ident (see [here](https://github.com/habitat-sh/core-plans/pull/2368/files#diff-aa3030b2fb8a2e702035bfe82a90f054R1))

It's not ideal because it may need repetitive test code. I was hoping `pkg_ident` would be in the environment; but it seems it isn't when InSpec is running.